closed/Intel/code/resnet50/pytorch-cpu/run_offline.sh

```diff
@@ -30,30 +41,31 @@ if [ -z "${RN50_FULL}" ]; then
     exit 1
 fi
 
-CONDA_ENV_NAME=rn50-mlperf
-source ~/anaconda3/etc/profile.d/conda.sh
-conda activate ${CONDA_ENV_NAME}
+#CONDA_ENV_NAME=rn50-mlperf
+#source ~/anaconda3/etc/profile.d/conda.sh
+#conda activate ${CONDA_ENV_NAME}
 
 export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:9000000000,muzzy_decay_ms:9000000000"
 
-export LD_PRELOAD=${CONDA_PREFIX}/lib/libjemalloc.so
+#export LD_PRELOAD=${CONDA_PREFIX}/lib/libjemalloc.so
 
-export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libiomp5.so
+#export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libiomp5.so
```

Comment:
if we could bring the line back, it should help on perf

Reply:
Done.